studio/data/config/project-api-query.ts

```diff
@@ -51,7 +51,7 @@ export type AutoApiService = Service & {
   protocol: 'https' | 'http'
   endpoint: string
   restUrl: string
-  project: Project
+  project: Pick<Project, "ref">
```

Comment:
A bit strange if only ref is returned for this project. Need to double check on platform